examples/ansible/setup_kea_dhcp_server.yml

```diff
@@ -0,0 +1,87 @@
+---
+# Ansible Playbook: Manage Kea DHCP Server Workload on ALP Host
+# Description: This Ansible playbook automates the setup of the Kea DHCP server workload on an ALP host. It follows the steps documented in the URL provided below.
+# Kea Workload Documentation: https://build.opensuse.org/package/view_file/SUSE:ALP:Workloads/kea-container/README.md?expand=1
+
+# Playbook Variables:
+#   - 'run_dhcpv6': A boolean variable to control the Kea DHCPv6 server. Set to 'true' to start the DHCPv6 server, 'false' to skip it.
+#   - 'dhcp4_template_src': Path to the DHCPv4 configuration j2 template. Leave empty if not using a template.
+#   - 'dhcp6_template_src': Path to the DHCPv6 configuration j2 template. Leave empty if not using a template.
+#   - 'kea_config_path': Path where the Kea DHCP server configuration files will be stored.
+#   - 'workload': A dictionary containing information about the Kea workload, including its name and container image.
+
+- name: Deploying and Managing the Kea DHCP server workload
+  hosts: alphost
+  become: true
+  vars:
+    run_dhcpv6: false
+    dhcp4_template_src: ""
+    dhcp6_template_src: ""
+    kea_config_path: /etc/kea
```

Comment:
Why is this even configurable? And if I decide that `kea_config_path: /opt/dhcp` will it the daemon pick?